src/set_theory/game/ordinal.lean

```diff
@@ -157,8 +174,67 @@ theorem to_pgame_add : ∀ a b : ordinal.{u}, a.to_pgame + b.to_pgame ≈ (a ♯
 end
 using_well_founded { dec_tac := `[solve_by_elim [psigma.lex.left, psigma.lex.right]] }
 
-@[simp] theorem to_pgame_add_mk (a b : ordinal) :
-  ⟦a.to_pgame⟧ + ⟦b.to_pgame⟧ = ⟦(a ♯ b).to_pgame⟧ :=
+/-! ### Ordinals to `game` -/
+
+/-- Converts an `ordinal` into a `pgame`. -/
+noncomputable def to_game : ordinal ↪o game :=
+{ to_fun := λ o, ⟦o.to_pgame⟧,
+  inj' := λ a b h, by rwa [←equiv_iff_game_eq, to_pgame_equiv_iff] at h,
+  map_rel_iff' := @to_pgame_le_iff }
+
+@[simp] theorem to_game_def (o : ordinal) : ⟦o.to_pgame⟧ = o.to_game := rfl
+
+theorem to_game_strict_mono : strict_mono to_game :=
+to_pgame_strict_mono
+
+@[simp] theorem zero_to_game : to_game 0 = 0 :=
+quot.sound zero_to_pgame_equiv
```

Comment:
This should be called `to_game_zero`, same throughout

Reply:
`zero_to_pgame` would also need to be renamed. I thought that for functions `to_y` we could use the names `x_to_y`?